front/tests/global-setup.ts

```diff
@@ -0,0 +1,16 @@
+import { exec } from 'child_process';
+
+export default async function globalSetup() {
+  exec(
+    'poetry --directory ../python/railjson_generator/ run python -m railjson_generator ./tests/assets/ small_infra',
```

Comment:
```suggestion
    'pwd',
```
just to see where is executed this command ...